src/main/java/pl/poznan/put/cs/scenariochecker/controller/ScenarioController.java

```diff
@@ -32,4 +33,14 @@ public String count() {
         this.scenarioStrategy.processScenario(this.scenario);
         return String.valueOf(this.scenario.getNumberOfSteps());
     }
-}
\ No newline at end of file
+
+    @GetMapping("/countSpecialSteps")
```

Comment:
I am not 100% sure if camel case is a good practice in urls, what do you think about `/steps/quantity` ?